src/util/gentle-register-property.js

```diff
@@ -18,6 +16,34 @@ const INITIAL_VALUES = {
 	"<url>": "url('')",
 };
 
+/**
+ * All registered properties.
+ * @type { Set<string> }
+ * @private
+ */
+let _properties = new Set();
```

Comment:
Why do all of these start with underscores? If they're not exported they're private anyway.